.github/workflows/terraform-plan-apply.yml

```diff
@@ -22,8 +30,15 @@ jobs:
         run: |
           git fetch origin main:localmain
 
-          # Gets changed json files excluding the json schema
-          export FILES=$(git diff --name-only localmain | grep .json | grep -v "trial-json-schema")
+          if [ ! -z ${{ github.event.inputs.env }} ]
+          then
+            echo "::set-output name=deploy_env::$( echo ${{github.event.inputs.env}} )"
+            export FILES=$(git diff --name-only localmain | grep .json | grep "\/${{ github.event.inputs.env }}_.")
```

Comment:
I think both here and on line 40, we should change the grep to
definition.json
and definition_env.json

meaning not just any .json file. 
at first we thought any file would do, but I think a constant name is better.
this will also allow us to remove the -v part

Reply:
when we wrote this there was also the config.json to consider..as we now look like using yaml we could change this...but if so I do think that we should have another story for that work.